### ISSUE TITLE
Disable formatting for a file extension

### ISSUE THREAD

**Issue by phillip-haydon** [Glavin001/atom-beautify#848]
### Description

I expected the indentation of a `.tag` file to be correct, however it actually causes everything to end up on 1 line. 

I simply want to just turn off support for `.tag` but don't know how.

### Expected Results

The beautified code should have looked like:

```
<help-video>

    <div>
        <iframe src="{ opts.video }"
                width="1279" height="720" frameborder="0"
                webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
    </div>

    <script type="babel">
        this.root.addEventListener('click', e => {
            if (e.target === this.root) {
                document.documentElement.removeChild(this.root);
            }
        });
    </script>
    
</help-video>
```

But it actually looks like:

```
<help-video>

    <div>
        <iframe src="{ opts.video }" width="1279" height="720" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
    </div>

    <script type="babel">
        this.root.addEventListener('click', e => { if (e.target === this.root) { document.documentElement.removeChild(this.root); } });
    </script>

</help-video>
```

### Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Checklist

- [X] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [X] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues 
so I know this is not a duplicate issue
- [ ] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by prettydiff** [comment]
Would you mind providing a gist to a debug.md?  It sounds like a valid bug that needs to be fixed.  I believe this is something that would be processed by Pretty Diff, but when I run this through Pretty Diff (with sorting and wrapping) I get:

    <help-video>

        <div>
            <iframe
                allowfullscreen
                frameborder="0"
                height="720"
                mozallowfullscreen
                src="{ opts.video }"
                webkitallowfullscreen
                width="1279"></iframe>
        </div>

        <script type="babel">
            this.root.addEventListener('click', e => { if (e.target === this.root) {
            document.documentElement.removeChild(this.root); } });
        </script>

    </help-video>

**Comment by phillip-haydon** [comment]
https://gist.github.com/phillip-haydon/237a82f2a4925cacdb68

I hope I've done that correctly.

**Comment by prettydiff** [comment]
@phillip-haydon I don't mean to sound picky, but if you could change the file format of that gist from `.txt` to `.md` it would really increase readability.

**Comment by phillip-haydon** [comment]
You're not being picky. Didn't even occur to me me was markdown. 

Fixed now, sorry about that.

**Comment by prettydiff** [comment]
It looks like the `html_wrap_line_length` option is provided a value of **250**.  Nothing in the code sample is that many characters wide, so nothing wraps.  Pretty Diff will indent each and every attribute if a tag is longer than the above mentioned value.  The code sample I posted above is because I used the default value of `80`.

If you want attributes to wrap set the value for this option to a smaller number.  If you only want some attributes to wrap so that the tag flows like content instead of indenting each and every attribute I would have to do some work to create a new option and modify how attributes respond to line wrapping.

**Comment by phillip-haydon** [comment]
The default is 250...

![default](https://cloud.githubusercontent.com/assets/895629/13547428/413c69dc-e30d-11e5-97a0-e105ad06287d.png)

**Comment by prettydiff** [comment]
Try it at 80 and let me know if that works better for you.

**Comment by phillip-haydon** [comment]
:S it's even worse:

https://gist.github.com/phillip-haydon/a4fcce6c2ecfdcae7597

Just turns JS into a giant block of text...

BUT I know what the problem is...

`<script type="babel">`

If I update it to: `<script type="text/javascript">` it works fine...

It doesn't handle babel :(

**Comment by prettydiff** [comment]
Thanks for identifying that.  That will be an easy fix on my end.  I will attempt to publish an update this evening.  Once the fix is published I will update and close this issue.

**Comment by phillip-haydon** [comment]
:+1: awesome, thanks a lot :heart:

**Comment by prettydiff** [comment]
Release is pending resolution of issue https://github.com/prettydiff/prettydiff/issues/285

**Comment by prettydiff** [comment]
Released with version v1.16.28.

**Comment by phillip-haydon** [comment]
@prettydiff it's all working perfectly now. Thanks a lot. Sorry for late update. Just wanted you to know I appreciate the fix :)
